handbook/company/communications.md

```diff
@@ -229,31 +265,48 @@ Here are some tips for troubleshooting Gong:
 - To counter this, Nathan Holliday will periodically delete all recordings found in Zoom's storage without viewing them.
 - If you need help using Gong, please check out [Gong Academy](https://academy.gong.io/).
 
+
 ### Excluding calls from being recorded
+
 For those with a Gong seat or scheduling a call with someone in attendance that has a Gong seat who does not wish for their Zoom call with an external party to record, make sure your calendar event title contains `[no shadows]`.  You can also read the [complete list of exclusion rules](https://docs.google.com/document/d/1OOxLajvqf-on5I8viN7k6aCzqEWS2B24_mE47OefutE/edit?usp=sharing).
 
 
 ## Levels of confidentiality
+
 Fleet uses these levels to standardize a commitment to minimal esotericism across the company.  
 - **Public:**   _Share with anyone, anywhere in the world_
 - **Confidential:**  _Share only with team members who've signed an NDA, consulting agreement, or employment agreement_
 - **Classified:**  _Share only with founders of Fleet, business operations, and/or the people involved.  e.g., US social security numbers during hiring_
 
+
 ### Document titles
+
 Fleet uses these levels to standardize a commitment to minimal esotericism across the company. 
 - **"Public":** _(Available to public)_ 
 - _(Confidential - for Fleet eyes only)_     
 - **"¶":** _(E-group - Direct reports the the CEO)_
 - **"¶¶":** _(Classified - CEO, Apprentice, and BizOps)_
 - **"¶¶¶":** _(CEO, Apprentice to the CEO, and board members)_
 
+## Google Drive
+
+Fleet uses Google Drive in order to keep files correctly bucketed and as accesable as possible for all team members. Here are some best practices for to keep in mind:
+- Check for existing files defore uploading to avoid duplication.
+- Avoid breaking google drive links when uploading new versions of existing files.  In most cases, you can update files by using the "Manage versions" option (in the right click menu) so links continue to work.
```

Comment:
```suggestion
- Avoid breaking Google Drive links. In some cases, you can update files by using the "Manage versions" option (in the right-click menu) so that links continue to work.
```